packages/js/src/generators/library/library.ts

```diff
@@ -5,13 +5,7 @@ import { GeneratorSchema } from '../../utils/schema';
 
 export async function libraryGenerator(tree: Tree, schema: GeneratorSchema) {
   const { libsDir } = getWorkspaceLayout(tree);
-  return projectGenerator(
-    tree,
-    schema,
-    libsDir,
-    join(__dirname, './files'),
-    'library'
-  );
+  return projectGenerator(tree, schema, libsDir, join(__dirname, './files'));
```

Comment:
nitpick: `projectGenerator` was here because we reuse it for `library` and `application`. Now that it's only `library`, should we still keep this abstraction?

Reply:
Good point, we should inline it in the library generator.